dashboard/app/models/levels/poetry.rb

```diff
@@ -33,11 +33,11 @@ class Poetry < GamelabJr
     available_poems
   )
 
-  # Set the default poem to nil if the subtype does not have poems, or if the default poem is
+  # Set the default poem to nil if the subtype, i.e., standalone_app, does not have poems, or if the default poem is
   # not in the list of poems for the subtype.
   def sanitize_default_poem
     self.default_poem = nil if Poetry.subtypes_with_poems.exclude?(standalone_app_name) ||
-      Poetry.poem_keys_for_subtype(standalone_app_name).exclude?(default_poem)
+      Poetry.poem_list_keys_for_standalone_app(standalone_app_name).exclude?(default_poem)
```

Comment:
nit: I think it would be a bit cleaner to name these `poem_keys_for_standalone_app` and `poems_for_standalone_app`, because a plural implies a list.

Reply:
Sure - sounds good. Will update

Reply:
And just to clarify, did you mean `poems_keys_for_standalone_app` or `poem_keys_for_standalone_app`?